Description: Add a autogen.sh script to easy the configure regeneration.
Author: Benjamin Drung <bdrung@debian.org>
Applied-Upstream: 1.3.13

--- /dev/null
+++ b/autogen.sh
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+aclocal -I m4
+autoconf
+rm -rf aclocal.m4 autom4te.cache
